nautobot_floor_plan/views.py

```diff
@@ -28,10 +37,22 @@ class LocationFloorPlanTab(ObjectView):
     template_name = "nautobot_floor_plan/location_floor_plan.html"
 
 
-class FloorPlanTileUIViewSet(NautobotUIViewSet):  # TODO we only need a subset of views
+class FloorPlanTileUIViewSet(
+    ObjectListViewMixin,
+    ObjectDetailViewMixin,
+    ObjectEditViewMixin,
+    ObjectDestroyViewMixin,
+    ObjectChangeLogViewMixin,
+    ObjectNotesViewMixin,
+    ObjectPermissionRequiredMixin,
```

Comment:
Is this mixin needed? I would have thought the viewset mixins would already enforce appropriate permissions.

Reply:
@glennmatthews 

Great question.  I am not sure about the `ObjectPermissionRequiredMixin` mixin.  I likely saw it referenced somewhere else and followed suit.  It basically just needs the same as NautobotUIViewSet minus the bulk mixins.  

Referencing that from core, I believe it can safely be removed:

```python
class NautobotUIViewSet(
    mixins.ObjectDetailViewMixin,
    mixins.ObjectListViewMixin,
    mixins.ObjectEditViewMixin,
    mixins.ObjectDestroyViewMixin,
    #mixins.ObjectBulkDestroyViewMixin,
    #mixins.ObjectBulkCreateViewMixin,  # 3.0 TODO: remove, no longer needed after ImportObjects system Job.
    #mixins.ObjectBulkUpdateViewMixin,
    mixins.ObjectChangeLogViewMixin,
    mixins.ObjectNotesViewMixin,
):
```